php_fpm/README.md

```diff
@@ -40,6 +40,52 @@ Configuration Options:
 
 [Restart the Agent][3] to start sending PHP-FPM metrics to Datadog.
 
+#### Multiple pools
+
+It is also possible to monitor multiple PHP-FPM pools using the same proxy server, a common scenario when running on Kubernetes.
+
+To do so, modify each instance's `php-fpm.conf`. Here is an example:
+
+Instance 1
+
+```
+...
+pm.status_path = /status1
+ping.path = /ping1
+...
+```
+
+Instance 2
+
+```
+...
+pm.status_path = /status2
+ping.path = /ping2
+...
+```
+
+Then you must modify your server's routes. Here is an example Nginx configuration:
```

Comment:
r / `Here is an example Nginx configuration:` / `Here is an example configuration for the Nginx integration`:

Reply:
This is for Nginx itself, not our integration.

Reply:
duly noted, then this comment is irrelevant.